### DISCUSSION TITLE
Ölzähler virtuell

### DISCUSSION THREAD

**Discussion by kalleick** [dewenni/ESP_Buderus_KM271#62] [Q&A] [answered]
Wie kann man den Parameter für Verbrauch Kg/h am besten ermitteln?
Wir haben heute 2000 Liter Öl geliefert bekommen und die virtuellen Verbrauchswerte, weichen rückblickend ab. 
Wie "justiere" ich diesen virtuellen Ölzähler am besten, einfach über Dreisatz?

**Comment by the78mole** [comment, marked as answer]
Hallo Kalleick,

zwei Anmerkungen/Rückfragen zu Deiner Frage:

Hast Du die Dichte korrekt eingestellt? Laut [Datenblatt](https://www.bp.com/content/dam/bp/country-sites/fr_ch/switzerland/home/produits-et-services/fuer-den-grosshandel/hel_euro_qualitaet_v2_1_01_03_2013_wos.pdf) hat Heizöl eine Dichte von 820-860 kg/m³. Liegt Deine Abweichung bereits außerhalb dieser Toleranzschwelle?

Hast Du die Standard-Brennerdüse für Deinen Kessel verbaut? Die bestimmt im wesentlichen den Ölverbrauch pro Stunde. Bei meinem 21 kW-Brenner ist eine Düse verbaut, die ca. 2,1 Liter/h "durchlässt". 

Ansonsten liegst Du völlig richtig, einfach das Verhältnis von "gemessenem" Verbrauch und gelieferter Menge ermitteln und die Verbrauchskonstante damit korrigieren. 

Grüße,
Daniel

**Comment by kalleick** [comment]
Hallo Daniel, danke für die super schnelle Antwort. Laut Wikipedia liegt die Dichte bei Heizöl bei 0,820–0,860 kg/l (15 °C).  Demnach habe ich den Wert auf die Mitte (0.8400 kg/l) eingestellt. Den Verbrauch hatte ich auf 1.55 kg/h gestellt, vermutlich ist das zu wenig. Ich habe den Brenner Buderus BE-A 1.1-17K 18-21,5 kW - eben mal im Datenblatt geschaut, er hat einen Öldurchsatz zwischen 1,5 und 1,8 kg/h. Jetzt erinnere ich mich auch wieder, woher ich den Wert genommen hatte. Trotzdem scheint am Ende irgendwas nicht zu passen, ich rechne nochmals alles durch und passe die Konstante an. Vielen Dank
